db/patterns/danawa.eno

```diff
@@ -0,0 +1,14 @@
+name: Danawa
+category: advertising
+website_url: https://danawa.com/
+organization: connectwave
+
+--- domains
+ad.danawa.com
+las.danawa.com
+--- domains
+
+--- filters
+||ad.danawa.com
```

Comment:
If I go to the domain (https://ad.danawa.com/), it does an opt-out. Would be an argument in favor of triggering only for $3p requests (`||ad.danawa.com$3p`)

Reply:
Fixed.